Formula/go-kubectx.rb

```diff
@@ -0,0 +1,24 @@
+class GoKubectx < Formula
+  desc "Go-based kubectx alternative. 5x-10x faster"
+  homepage "https://github.com/aca/go-kubectx"
+  url "https://github.com/aca/go-kubectx/archive/v0.1.0.tar.gz"
+  sha256 "9bcb7797a2581fd240a12141a129a8e62997ddc32ba75b4973563cafb995db2a"
+  head "https://github.com/aca/go-kubectx.git"
+
+  depends_on "go" => :build
+  depends_on "fzf"
+
+  conflicts_with "kubectx", :because => "kubectx also ships a kubectx binary"
+
+  def install
+    Dir.chdir "cmd/kubectx"
+    system "go", "build", "-o", "#{bin}/kubectx"
+    Dir.chdir "../kubens"
+    system "go", "build", "-o", "#{bin}/kubens"
+  end
+
+  test do
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I took the test from the standard `kubectx` config :)

https://github.com/Homebrew/homebrew-core/blob/4f69a8b8be19a6d36942a074e131b434959a710a/Formula/kubectx.rb#L23-L26

Reply:
> I took the test from the standard `kubectx` config :)

Then that one is also bad. We hold all new submissions to the same standard, it needs to have a good test. Maybe you can make a pull request to test the old kubectx better as well?

Reply:
Yeah, this will be basically the same test. I assume I can create files, etc in that routine? Should I remove them at the end of the test?

Reply:
Yeah you can. It's a temporary directory so no need to clean up.